runtime_util/runtime_util.h

```diff
@@ -31,9 +32,10 @@ typedef enum gp_joysticks {
 	X_LEFT_JOYSTICK, Y_LEFT_JOYSTICK, X_RIGHT_JOYSTICK, Y_RIGHT_JOYSTICK
 } gp_joystick_t;
 
+
 //enumerated names for the different values the robot description fields can take on
 typedef enum robot_desc_vals {
-	IDLE, AUTO, TELEOP,         //values for robot.run_mode
+	IDLE, AUTO, TELEOP, ESTOP,       //values for robot.run_mode
```

Comment:
Did we end up needing the `ESTOP` value? I don't see it anywhere in your code so I'm assuming no? If we don't need it we should delete it~~

Reply:
Executor wouldn't be using it, only the supervisor would.